modules/ROOT/pages/ps-create-configure.adoc

```diff
@@ -0,0 +1,370 @@
+to deploy your apps to.
+
+
+== Create a Private Space 
+
+//SELECT PRIVATE SPACE SHARED
+include::partial$select-private-space.adoc[tag=selectPrivateSpace]
+. Click *Create private space*.
+. Enter a xref:ps-gather-setup-info.adoc#name[name] for your private space and click *Create*.
+
+
+== Create a Private Network
+
+[IMPORTANT]
+You can't change the private network configuration after it's created.
+To change the configuration, delete the private network and create it again.
+
+//SELECT PRIVATE SPACE SHARED
+include::partial$select-private-space.adoc[tag=selectPrivateSpace]
+include::partial$select-private-space.adoc[tag=clickPrivateSpaceName]
+. In the *Private network* tab, click *Create Private Network*. 
+. In the *Create Private Network* page:
+.. Select the xref:ps-gather-setup-info.adoc#region[region] where you want your apps to run.
+.. Enter the xref:ps-gather-setup-info.adoc#cidr-block[CIDR block], which is the range of IP addresses that your apps can use.
+.. Click *Create*.
++
+The private network can take up to 30 minutes to create.
++
+While the network is created, you can 
+<<Create a Connection to an External Network,create a connection to an external network>>.
+. After the network creation succeeds, xref:ch2-deploy.adoc[deploy an application] to test the private network.
+
+If the private network creation fails, delete it and click *Create Private Network* to try again.
+
+=== Resolve Private Domains in Your Internal Network
+
+If your corporate network uses internal DNS servers to resolve requests to custom domains, configure the private space with these IP addresses and domain names.
+The private space uses your internal DNS to resolve internal host names of your private network (make sure your applications call the backend resources by FQDN).
+
+//SELECT PRIVATE SPACE SHARED
+include::partial$select-private-space.adoc[tag=selectPrivateSpace]
+include::partial$select-private-space.adoc[tag=clickPrivateSpaceName]
+. In the *Private network* tab, click the *Configure* link for *Internal DNS servers*.
+. Enter the (comma-separated) xref:ps-gather-setup-info.adoc#server-ip-addresses[server IP addresses] for your internal DNS servers.
+. Enter the (comma-separated) xref:ps-gather-setup-info.adoc#domains[domain addresses] to be resolved by internal DNS.
+. Click *Create*.
+
+After you configure internal DNS, the *Private network* tab indicates the number of server IP addresses and domains:
+
+image::internal-dns.png[Internal DNS configuration in the Private Network section]
+
+== Create a Connection to an External Network
+
+After you create your private network, connect it to an external network using a 
+VPN or transit gateway attachment.
+
+//SELECT PRIVATE SPACE SHARED
+include::partial$select-private-space.adoc[tag=selectAppsPage]
+. In the *Private network* tab, click *Create Connection*. 
+. In the *Create Connection* page:
++
+--
+.. Enter a name for your connection.
```

Comment:
In the latest design, selecting the connection type comes before naming the connection. This is because if they select the TGW type, they have an opportunity to select any existing TGWs used in other private spaces first. If they do so, then there's no need to supply a name. That doesn't all necessarily need to be explained here though.

Reply:
Updated to match UI